src/OrderFormAcceptanceTests.Actions/Pages/PreviewOrderSummary.cs

```diff
@@ -158,5 +158,11 @@ public string GetTotalOwnershipCost()
         {
             return Driver.FindElement(Pages.PreviewOrderSummary.TotalOwnershipCost).Text;
         }
+
+        public bool ServiceInstanceIdColumn()
+        {
+            return Driver.FindElement(Pages.PreviewOrderSummary.RecurringCostsTable)
+                .FindElement(Pages.PreviewOrderSummary.ServiceInstanceIdColumn).Text.Equals("Service Instance ID", StringComparison.OrdinalIgnoreCase);
```

Comment:
Might it be worth doing a `StringComparison.Ordinal` comparison here? (Just trying to think from a content point-of-view.) What do you think?

Reply:
Not a bad idea, I think I went on autopilot with the ignore case bit....